sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/mapping/CosmosIndexingPolicy.java

```diff
@@ -13,13 +13,21 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation for cosmos indexing policy
+ * Annotation for cosmos indexing policy.
+ * Using this annotation will overwrite the cosmos indexing policy currently in Portal,
+ * unless the policy are identical or overwritePolicy is set to false.
```

Comment:
```suggestion
 * To prevent overwriting the Indexing Policy defined on Azure Portal, Indexing Policy defined on the SDK through this annotation should be identical or set overwritePolicy flag to false.
```